client/components/common/mixins/publish.vue

```diff
@@ -26,8 +27,10 @@ export default {
       this.isPublishing = true;
       return Promise.each(activities, activity => {
         this.publishStatus = `Publishing ${activity.data.name}`;
-        return this.publishActivity(activity);
+        return this.publishActivity(activity)
+          .then(() => this.activityNum++);
       }).then(() => {
+        this.activityNum = 0;
```

Comment:
Ok, but please put this (and following lines too) in `finally` block instead to make sure this gets executed on failure too. :bowtie: